composer.json

```diff
@@ -30,7 +30,8 @@
         "drupal/password_policy": "^3.0@alpha",
         "drupal/views_data_export": "^1.0",
         "drupal/hsts": "^1.0",
-        "consolidation/robo": "~1.2.4"
+        "consolidation/robo": "~1.2.4",
+        "dompdf/dompdf": "~0.8.2"
```

Comment:
A lot of stuff changed in the composer.lock file. Did you maybe do a "composer update"? My recommendation would be to revert all changes to composer.json and composer.lock, and then on the command line do a "composer require dompdf/dompdf".